pytest.ini

```diff
@@ -20,5 +20,7 @@ filterwarnings =
     ignore:.*will be removed in Pillow 10:DeprecationWarning:
     # For packages importing distutils in py 3.10 (tensorboard)
     ignore:.*distutils package is deprecated and slated:DeprecationWarning
+    # For warnings from torch 1.13
+    ignore:'torch.onnx._patch_torch._graph_op' is deprecated in version 1.13 and will be removed in version 1.14.*:FutureWarning
```

Comment:
Warning caused by `torch.C.Graph#op` is quite annoying so ignoring by this level